src/main/java/org/jabref/logic/util/CoarseChangeFilter.java

```diff
@@ -53,19 +42,30 @@ public synchronized void listen(BibDatabaseContextChangedEvent event) {
             // If editing is started
             boolean isNewEdit = lastFieldChanged.isEmpty();
             // If other field is edited
-            boolean isEditOnOtherField = !isNewEdit && !lastFieldChanged.get().equals(fieldChange.getField());
+            boolean isEditChanged = !isNewEdit && !lastFieldChanged.get().equals(fieldChange.getField());
             // Only deltas of 1 registered by fieldChange, major change means editing much content
-            boolean isMajorChange = totalDelta >= 100;
-
-            if ((isEditOnOtherField && !isNewEdit) || isMajorChange) {
-                // Submit old changes immediately
-                eventPost.run();
-            } else {
-                delayPost.schedule(eventPost);
-            }
+            boolean isMajorChange = totalDelta >= 30;
+
+            // Event is filtered out if neither the edited field has changed nor a major change has occurred
+            fieldChange.setFilteredOut(!(isEditChanged || isMajorChange));
+            // Post every FieldChangedEvent, but some have been marked (filtered)
+            eventPost(fieldChange);
             // Set new last field
             lastFieldChanged = Optional.of(fieldChange.getField());
+
         }
+        else {
+            eventPost(event);
+        }
+    }
+
+    private void eventPost(BibDatabaseContextChangedEvent event) {
+        // Reset total change delta
+        totalDelta = 0;
+        // Reset last field that changed
+        lastFieldChanged = Optional.empty();
+        // Post event
```

Comment:
This comment can be deleted, because it is just stating what happens in the next line